github/repos_contents_test.go

```diff
@@ -55,11 +55,19 @@ func TestRepositoryContent_GetContent(t *testing.T) {
 	for _, tt := range tests {
 		r := RepositoryContent{Encoding: tt.encoding, Content: tt.content}
 		got, err := r.GetContent()
+		stringOrNil := func(s *string) string {
+			if s == nil {
+				return "<nil>"
+			}
+			return *s
+		}
+		encoding := stringOrNil(tt.encoding)
+		content := stringOrNil(tt.content)
```

Comment:
I don't think it's appropriate to add this much code in between `got, err := ...` and the `if err != nil ...` statements while keeping the variable name just `err` (see [here](https://dmitri.shuralyov.com/idiomatic-go#error-variable-naming)). A future reader may think there's a missing error check after the `got, err := ...` line.

I would suggest factoring out the definition of `stringOrNil` to the very top of `TestRepositoryContent_GetContent`, or even outside to the package scope, below `TestRepositoryContent_GetContent`. And the `stringOrNil(...)` calls can just be inlined into the corresponding `t.Errorf`.

Reply:
agree, done with 2c7b329